.travis.yml

```diff
@@ -1,28 +1,51 @@
 language: julia
 os:
-- linux
+  - linux
 julia:
-- 1.0
+  - 1.2
 notifications:
   email: false
 git:
   depth: 99999999
 cache:
   timeout: 1000
   directories:
-  - downloads
-  - shards
+    - downloads
+    - shards
 env:
   global:
-  - BINARYBUILDER_DOWNLOADS_CACHE=downloads
-  - BINARYBUILDER_SHARDS_DIR=shards
-  - BINARYBUILDER_AUTOMATIC_APPLE=true
-  - BINARYBUILDER_USE_CCACHE=true
+    - BINARYBUILDER_DOWNLOADS_CACHE=downloads
+    - BINARYBUILDER_SHARDS_DIR=shards
+    - BINARYBUILDER_AUTOMATIC_APPLE=true
+    - BINARYBUILDER_USE_CCACHE=true
+  matrix:
+    - TARGET=i686-linux-gnu
+    - TARGET=x86_64-linux-gnu
+    - TARGET=aarch64-linux-gnu
+    - TARGET=arm-linux-gnueabihf
+    - TARGET=powerpc64le-linux-gnu
+    - TARGET=i686-linux-musl
+    - TARGET=x86_64-linux-musl
+    - TARGET=aarch64-linux-musl
+    - TARGET=arm-linux-musleabihf
+    - TARGET=x86_64-apple-darwin14
+    - TARGET=x86_64-unknown-freebsd11.1
+    - TARGET=i686-w64-mingw32-gcc7
+    - TARGET=x86_64-w64-mingw32-gcc7
```

Comment:
Is gcc7 necessary? :thinking:

Reply:
Ah probably not..was just copy and pasting this... Maybe we should come up with a more reasonable list?!

Reply:
Like the default one? :stuck_out_tongue: 
```julia
julia> using BinaryBuilder

julia> triplet.(supported_platforms())
13-element Array{String,1}:
 "i686-linux-gnu"            
 "x86_64-linux-gnu"          
 "aarch64-linux-gnu"         
 "arm-linux-gnueabihf"       
 "powerpc64le-linux-gnu"     
 "i686-linux-musl"           
 "x86_64-linux-musl"         
 "aarch64-linux-musl"        
 "arm-linux-musleabihf"      
 "x86_64-apple-darwin14"     
 "x86_64-unknown-freebsd11.1"
 "i686-w64-mingw32"          
 "x86_64-w64-mingw32"
```